AutomatedTesting/Gem/PythonTests/Atom/atom_utils/atom_constants.py

```diff
@@ -524,6 +526,23 @@ def grid(property: str = 'name') -> str:
         }
         return properties[property]
 
+    @staticmethod
+    def hair(property: str = 'name') -> str:
+        """
+        Atom Hair component properties. Requires Actor component.
+          - 'requires' a list of component names as strings required by this component.
+            Use editor_entity_utils EditorEntity.add_components(list) to add this list of requirements.
+          - 'Hair Asset'
```

Comment:
Need text description describing what "Hair Asset" is

Reply:
fixed